CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+## How to contribute
+
+We're really glad you're reading this, because we need volunteer developers to help this project come to fruition.
+
+The following is a set of guidelines for contributing to shadow service, which are hosted in the [Darwinia Network][0] Organization on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+
+## Tests
+
+We have test cases for Rust and Golang, you can trigger the test cases of Rust using `cargo test --all`, for Golang is `go test -v ./...`.
+
+
+### Optional Tests
+
+Here is a special cargo feature `darwinia` only for test usages, if you are contributing to the `scale codec` part of our rust code, you can run `cargo test --all-features` to trigger it.
+
+
+## Branches
+
+Here are 3 branches on this project
+
++ `v1`: The is the stable code which can be importing by other go projects
++ `next`: This is the branch our prs will merge into
++ `gh-pages`: This branch serves the API docs of shadow
```

Comment:
I'm thinking of deleting the `master` branch, it's easy to mix up the meaning of `master` and `v1` or `stable`, and `next` is more clear.

Reply:
You are repo maintainer, so I'm generally ok if not using master, but I think it's all about convention. 

My conversion now is master for merging, feature branch for PR, tagging for release and stable